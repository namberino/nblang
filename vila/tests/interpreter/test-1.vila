print(2 * (3 / -"dummy"));
