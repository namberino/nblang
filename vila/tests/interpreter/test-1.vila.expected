Operand must be a number
On line 1
